internal/gqlclient/gqlclient.go

```diff
@@ -53,7 +55,11 @@ func NewWithOpts(url string, timeout time.Duration, opts ...graphql.ClientOption
 }
 
 func New(url string, timeout time.Duration) *Client {
-	return NewWithOpts(url, timeout, graphql.WithHTTPClient(retryhttp.DefaultClient.StandardClient()))
+	httpClient := api.NewHTTPClient()
+	if condition.InUnitTest() {
+		httpClient = http.DefaultClient
+	}
+	return NewWithOpts(url, timeout, graphql.WithHTTPClient(httpClient))
```

Comment:
For some reason, the `retryhttp` client is having trouble connecting with the `platform.activestate.testing.tld` URL in unit tests. I spent some time trying to determine why and then just decided to go with this fix.

Reply:
Wouldn't this be better addressed inside `api.NewHTTPClient()`?

Reply:
Yes, that's a much better place for it.